src/kademlia/dht_storage.cpp

```diff
@@ -83,17 +84,15 @@ namespace
 	struct dht_immutable_item
 	{
 		// the actual value
-		std::unique_ptr<char[]> value;
+		std::vector<char> value;
```

Comment:
changing this to a vector will make it use 16 more bytes (3 pointers instead of 1). the ``int`` for the size will fill out the padding caused by ``num_announcers``